deploy/sre-prometheus/100-node-filedescriptor-limits.PrometheusRule.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         severity: critical
         namespace: openshift-monitoring
       annotations:
-        message: "Kernel is predicted to exhaust file descriptors limit soon."
+        message: "Kernel is predicted to exhaust file descriptors limit soon. Link: https://github.com/openshift/ops-sop/blob/master/v4/alerts/NodeFileDescriptorLimit.md"
```

Comment:
link will be in a separate line. Also please move the link to inside `labels`
Ref -  https://github.com/openshift/ops-sop/blob/master/v4/alerts/KubeControllerManagerCrashloopingSRE.md